### ISSUE TITLE
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.

### ISSUE THREAD

**Issue by claudiodriussi** [Rajath4/BluetoothThermalPrinter#32]
**Describe the bug**
I can't upgrade my gradle version because i got:

```
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':bluetooth_thermal_printer' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
```

If I compile with a less recent version of gradle I get:

```
w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath
```

For now it works, but may be not i future. Can you solve the issue?

Many thanks,
Claudio Driussi

**Comment by eugene254-ship-it** [comment]
Was this issue solved. I have opted to use blue_thermal_printer ^1.2.3

**Comment by eugene254-ship-it** [comment]
Remember to refactor your build.gradle (Module: bluetooth_thermal_printer). The error probably arises from there.

**Comment by claudiodriussi** [comment]
> Was this issue solved. I have opted to use blue_thermal_printer ^1.2.3

Yes, I will try. But I solved with flutter_bluetooth_printer which raises a warning, but has all java source code so will be recompiled with the flutter app.

**Comment by collohdicey** [comment]
Had the same issue, I updated this part of my build.gradle to solve the error.
`dependencies {
        classpath 'com.android.tools.build:gradle:7.2.0'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }`
Before i had 
`dependencies {
        classpath 'com.android.tools.build:gradle:7.3.0'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }`
Somehow changing from ` gradle:7.3.0` to `gradle:7.2.0` fixes the issue if you already have `ext.kotlin_version` > `'1.5.20'`

**Comment by enriquejuan2** [comment]
Use the following with the latest gradle:

bluetooth_thermal_printer:
    git:
      url: https://github.com/Rajath4/BluetoothThermalPrinter
      ref: 7abaebaf191b9dee8399f72ea4777c95cc5e194c
